apps/finance/app/src/components/NewTransfer/Deposit.js

```diff
@@ -39,6 +39,19 @@ const TOKEN_ALLOWANCE_WEBSITE = 'https://tokenallowance.io/'
 
 const tokenAbi = [].concat(tokenBalanceOfAbi, tokenDecimalsAbi, tokenSymbolAbi)
 
+const renderBalanceForSelectedToken = selectedToken => {
+  const { decimals, loading, symbol, userBalance } = selectedToken.data
+  if (loading || !userBalance) {
+    return ''
+  }
+
+  return userBalance === -1
```

Comment:
I think this should be `'-1'`?

Reply:
😅 this is why I need a second pair of eyes 😄